mazerunner-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/NewSessionScenario.kt

```diff
@@ -50,7 +50,7 @@ internal class NewSessionScenario(config: Configuration,
             flushAllSessions()
             Log.d("Bugsnag - New", "Second session flushed")
             Thread.sleep(SLEEP_MS)
-            client.notifyBlocking(generateException())
+            client.notify(generateException())
```

Comment:
Is this now a non-blocking call? Seems like a pretty fundamental change so I just want to understand the impact of of it better.